### ISSUE TITLE
Cannot change port number in graphite-web.conf

### ISSUE THREAD

**Issue by dantate** [graphite-project/graphite-web#1675]
After the default broke my webapps by trying to take over / in the server, I tried changing the port to 81, but it does not open (not showing in netstat -lna). 

 <VirtualHost *:81>

Nothing obvious in the error logs.

**Comment by deniszh** [comment]
Hello @dantate,
There's no `graphite-web.conf` config exists for Graphite, unfortunately. If you want to change graphite-web port you need to do that in your webserver config.
Which webserver do you use, Apache?

**Comment by dantate** [comment]
yeah, the graphite-web.conf is the apache conf.d file.  strange it's not reporting an error, it seems like thatd be something obvious to log.

here's a snippit.  all i did was change 80 to 81.
<VirtualHost *:81>
    ServerName graphite-web
    DocumentRoot "/usr/share/graphite/webapp"
    ErrorLog /var/log/httpd/graphite-web-error.log
    CustomLog /var/log/httpd/graphite-web-access.log common

**Comment by deniszh** [comment]
You need to add "Listen 81" directive somewhere near "Listen 80". Usually it is in `/etc/apache2/ports.conf` file and restart Apache.
It's not directly Graphite-related though. :)

**Comment by dantate** [comment]
No such file on CentOS.  going to give up on it; too many problems.   It keeps misidentifying my system as an old DNS name anyway, that I can find no trace of.

**Comment by dantate** [comment]
thanks though

**Comment by deniszh** [comment]
On Centos this file is named `/etc/httpd/conf/httpd.conf`

**Comment by dantate** [comment]
this is actually part of httpd.conf via the include conf.d directive; so it's all the same.  Unfortunately I have other apps that run on the server, so having it monoplize *:80 is unacceptable.

Would you recommend i open a separate ticket for the fact that something, somewhere is misidentifying my server as a legacy DNS entry that doesn't exist anywhere anymore?

**Comment by deniszh** [comment]
Not really understand what you mean... Anyway, there is a ton of articles how to add change Apache port on the Internet, this question is not specific to Graphite, for example, http://www.cyberciti.biz/faq/fedora-rhel-centos-configure-httpd-listen-multipleports/

**Comment by dantate** [comment]
Yeah, but the problem is graphite fails to load (without an error message) if the port is changed to anything but 80.  That makes it oddly suspicious.

**Comment by deniszh** [comment]
If you change port to 81 in Virtualhost section - but didn't add "Listen 81" directive - Graphite (or any application which is under that Virtualhost section) will not load, that's intended behaviour. Please configure your Apache properly.
